package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2020 Tobias Maedel <openwrt@tbspace.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-rockchip
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
```

Comment:
It would be nicer(?) if you'd grab the generated tarball by Github
https://github.com/openwrt/packages/blob/master/utils/ttyd/Makefile#L14 (Example)

Reply:
Hm, maybe nicer in terms of speed and reliability. 
I have now changed it to GitHubs .tar.gz codeload. URL. 
Unfortunately that added an additional folder rename and a bit of shuffling around, but it is now done.